Contest_API.md

```diff
@@ -1337,13 +1440,20 @@ zip archive. These must be stored directly from the root of the zip
 file, i.e. there must not be extra directories (or files) added unless
 these are explicitly part of the submission content.
 
-#### POST and PUT submissions
+#### Submission capabilities 
```

Comment:
I'm not sure if I like renaming this sections (also above and below)

Reply:
I see your point, but I don't love using HTTP methods as section titles and think the document will be easier to scan if we have a standard pattern for each section that talks about any POST/PUT/PATCH/DELETE options. Maybe something like "Modifying submissions" (and e.g. "Modifying awards" in the future), or (since we have "Examples" for each type) leaving the type name out and use something like "Editing"?

Reply:
I like your last two suggestions way more

Reply:
I just did a commit with a "Modifying x" pattern, partially b/c if I went with a single word I don't know how to link to each section in markdown.  ;-)